src/app/components/history/history.component.html

```diff
@@ -1,3 +1,2 @@
-<p>
-  history works!
-</p>
+<ion-button id="clearHistoryBtn" size="small" color="danger" (click)="clearHistory()">Clear History</ion-button>
```

Comment:
when I click on the Clear History Button and then press OK, it doesn't clear the history.
Can you look into it plz? 

Else the history is saved and the UI is good.

Reply:
The clear button works as expected, it is just commented it out for the time being to test something. If you would like to see it in action just uncomment the lines in the clearHistory method body